index.css

```diff
@@ -1,4 +1,127 @@
 * {
     box-sizing: border-box;
+    list-style: none;
 }
 
+body {
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.wrapper {
+    overflow: auto;
+    width: 100vw;
+    height: 100vh;
+}
+
+section {
+    float: left;
+    height: 100%;
+    width: 50%;
+}
+section div.table, section div.images {
+    width: 90%;
+    height: 80%;
+    margin: 10% 5%;
+}
+.table {
+    border:solid black;
+    height: 100%;
+    overflow-y: scroll;
+}
+
+.row {
+    width: 100%;
+}
+
+ul {
+    padding-top: 3%;
+    border-top: solid black;
+    width: 100%;
+    text-align: center;
+    overflow: auto;
+}
+
+.heading {
+}
+
+li {
+    margin-left: 9%;
+    height: 10%;
+    width: 18%;
+    float: left;
+}
+ul li:first-child {
+    margin-left: -3%;
+width: 3%;
+}
+
+ul li:last-child {
+    width: 3%;
+}
+
+.images {
+    overflow-y: scroll;
+    float: right;
+    border: solid black;
+    height: 100%;
+    width: 100%;
+    position: relative;
+}
+.images p {
+    position: absolute;
+    color: white;
+    font-size: 1.5rem;
+}
+
+.images p:first-child {
+    position: absolute;
+}
+
+
+.images p:first-child, .images p:nth-child(4), .images p:nth-child(7), .images p:nth-child(10), .images p:nth-child(13) {
+    left: 5%;
+}
+
```

Comment:
have you considered using classes in place of many regularly repeating nth childs?
particularly if you mean to add more entires